src/main.c

```diff
@@ -2392,6 +2392,12 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
                 g->isWalking = false;     
 			} 
 		}
+        if (key == CRAFT_KEY_TIMETRAVEL) {
+            glfwSetTime(glfwGetTime() + 80);
```

Comment:
Code is easy enough to understand here, but I'd recommend setting the time interval to a constant variable and using that instead so the reader knows what 80 means.